### ISSUE TITLE
[!!] Enhance Your Brand bullet

### ISSUE THREAD

**Issue by slhenry** [w3c/wai-bcase#88]
Context:
> * Drive Innovation: Accessibility features in products and services often solve unanticipated problems.
>* Enhance Your Brand: Diversity and inclusion efforts so important to business success are accelerated with a clear, well-integrated accessibility commitment.
>* Extend Market Reach: The global market of people with disabilities is over 1 billion people with a spending power of more than $6 trillion. Accessibility often improves the online experience for all users.
>* Minimize Legal Risk: Many countries have laws requiring digital accessibility, and the issue is of increased legal concern.

This sentence doesn't make sense to me: "Diversity and inclusion efforts so important to business success are accelerated with a clear, well-integrated accessibility commitment."

Also, it doesn't seem parallel to the other bullets that **succinctly summarize the benefit or risk of accessibility**.

It seems like is should be a _shorter_ version of something like: Accessibility can significantly impact your organization's reputation. An accessible website can garner positive attention from people with disabilities and others. Some organizations actively promote accessibility as a business differentiator. Conversely, if an organization's website has accessibility problems, it can get negative publicity from news publications and on social media.

**Comment by slhenry** [comment]
I guess I just don't get it. Never mind.
